provider/vdpa_sim_utils.py

```diff
@@ -159,3 +159,32 @@ def add_dev(self, name, mac):
             raise VDPABlkNetSimulatorError(
                 "vdpa dev add %s failed:%s" % (name, str(e)))
         return dev
+
+
+class VirtioVdpaNetSimulatorTest(VDPABlkNetSimulatorTest):
+    def __init__(self):
+        super(VirtioVdpaNetSimulatorTest, self).__init__()
+        self._modules = ['vdpa', 'virtio-vdpa', 'vdpa_sim', 'vdpa-sim-net']
+
+    def add_dev(self, name, mac):
+        """
+        Add vDPA net device
+        :param name: device name
+        :param mac: MAC address
+        :return : host device name ,eg. eth0
+        """
+
+        cmd = "vdpa dev add name %s mgmtdev vdpasim_net mac %s" % (name, mac)
+        process.run(cmd, shell=True)
+        cmd = "vdpa dev list -jp %s" % name
+        process.run(cmd, shell=True)
+        time.sleep(2)
+        try:
+            virtio_dir = "ls /sys/bus/vdpa/devices/{}/virtio*/net".format(name)
+            s, o = process.getstatusoutput(virtio_dir, shell=True)
+            if s == 0:
+                dev = o
+            return dev
+        except Exception as e:
+            raise VDPABlkNetSimulatorError(
+                "vdpa dev add %s failed:%s" % (name, str(e)))
```

Comment:
```suggestion
        from glob import glob
        from aexpect.utils import wait
        virtio_dir = "/sys/bus/vdpa/devices/{}/virtio*/net".format(name)
        if not wait.wait_for(glob(virtio_dir), 2):
            raise VDPABlkNetSimulatorError
```

```python
>>> glob('/dev/kv*')
['/dev/kvm']
>>> glob('/sys/bus/vdpa/devices/*/virtio*/net')
[]
```

Reply:
It is need to return the device name rather than just checking if the directory exists.

Reply:
I have updated, please help review. thanks a lot.